cypress/integration/desktop/tapcheck.bathroom.cy.js

```diff
@@ -0,0 +1,76 @@
+// Primary Goal: Test each tap in without incurring heavy use of the Maps API, such as loading dynamic maps multiple times
+
+// - Builds a queue of all RESOURCE_TYPE locations
+// - While the queue is not empty, pulls an item from the queue and tests clicking it
+// - If the site crashes, records the tap that crashed, reloads the site, and continues looping
+
+const RESOURCE_TYPE = "BATHROOM"
+
+let exception_ocurred = false;
+let exception_message = "";
+
+// For each resource type, test each site detail permutation and confirm only the expected number of taps appear.
+describe('site info', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it(`should tap through all ${RESOURCE_TYPE} sites`, () => {
+    let test_target_queue = [5, 4, 3, 2, 1];
+    let test_target_failures = [];
+
+    cy.on('uncaught:exception', (err, runnable) => {
+      exception_message = err.message;
+      exception_ocurred = true;
+      return false;
+    })
+
+    const testLocation = () => {
+      const target_location_id = test_target_queue.pop()
+      cy.task('log', `Testing ${target_location_id}`)
+      cy.get(`[title=data-cy-${target_location_id}]`).click({ force: true, timeout: 8000 }).then(() => {
```

Comment:
Are we able to use a data-test-id instead of using title attributes? The title attribute is not intended for testing purposes

Reply:
We reviewed this on our weekly call, this seems to be a limitation of the Google Maps Marker Javascript class, which doesn't allow adding custom attributes.